setup.py

```diff
@@ -1,20 +1,20 @@
 #!/usr/bin/env python
 
-VERSION = '1.2.0'
```

Comment:
I am not sure that it is correct to move this down. It is good to have the version number at the top, because this is the only thing in this file which may be changed frequently. Maybe we should change this variable into a dunder variable instead, see https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names (but I have to read what a dunder variable actually is now...)

Reply:
`pycodestyle` complains:

    setup.py:5:1: E402 module level import not at top of file
    setup.py:6:1: E402 module level import not at top of file

Reply:
But if you add the double underscores before and after the variable, i.e. make it a _dunder variable_ `__VERSION__`, then `pycodestyle` is happy. I just don't know if there are any other effects by that.